notebooks/OrthoLearner.ipynb

```diff
@@ -0,0 +1,1126 @@
+{
```

Comment:
This doesn't look like a user-facing notebook - if it's just something that you're using for testing then don't commit it.

Reply:
I mostly committed it because I'm working from multiple workstations. Will remove it from the PR once all is finalized and right before merge.

Reply:
Removed the notebook.